src/juicy-popover.html

```diff
@@ -128,7 +128,7 @@
                 }
             }.bind(this));
 
-            expandable.addEventListener("mouseup", function (ev) {
```

Comment:
I really think it needs a listener on `expandableselector` since it has "ignore = false" every time `mouseup` is fired.
So two clicks in a row would produce the collapse if I remove this.